uisdk/src/main/java/com/karhoo/uisdk/screen/booking/quotes/capacity/CapacityView.kt

```diff
@@ -37,15 +37,15 @@ class CapacityView @JvmOverloads constructor(
         }
 
 
-        val capabilitiesCount = otherCapabilities?.minus(DEFAULT_CAPABILITIES_NO)
-
-        if (capabilitiesCount != null && capabilitiesCount > 0) {
-            otherCapabilitiesText.text =
-                resources.getString(R.string.kh_uisdk_extra_capabilities, capabilitiesCount)
-            otherCapabilitiesText.visibility = VISIBLE
-        } else {
-            otherCapabilitiesText.visibility = GONE
-        }
+//        val capabilitiesCount = otherCapabilities?.minus(DEFAULT_CAPABILITIES_NO)
+//
+//        if (capabilitiesCount != null && capabilitiesCount > 0) {
+//            otherCapabilitiesText.text =
+//                resources.getString(R.string.kh_uisdk_extra_capabilities, capabilitiesCount)
+//            otherCapabilitiesText.visibility = VISIBLE
+//        } else {
+//            otherCapabilitiesText.visibility = GONE
+//        }
```

Comment:
do we still need the other capabilities? I saw the otherCapabilitiesText is also gone now